docs/error-codes.md

```diff
@@ -245,3 +245,11 @@ error and warning codes.
 #### `IL2047`: DynamicallyAccessedMemberTypes in DynamicallyAccessedMembersAttribute on <member> don't match overridden <base member>. All overriden members must have the same DynamicallyAccessedMembersAttribute usage.
 
 - All overrides of a virtual method including the base method must have the same DynamicallyAccessedMemberAttribute usage on all it's components (return value, parameters and generic parameters).
+
+#### `IL2048`: Internal attribute RemoveAttributeInstances is being used on member that is not a type
```

Comment:
```suggestion
#### `IL2048`: Internal attribute RemoveAttributeInstances is being used on '<member>' that is not a type
```

Write out what it's used on - much easier to find and resolve - and we have the information anyway.